tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/files/S3StorageSpec.scala

```diff
@@ -108,14 +108,20 @@ class S3StorageSpec extends StorageSpec {
         .deepMerge(Json.obj("region" -> Json.fromString("eu-west-2")))
 
     for {
+      _ <- log("Creating first storage")
       _ <- storagesDsl.createStorage(payload, projectRef)
+      _ <- log("Created storage")
       _ <- storagesDsl.checkStorageMetadata(projectRef, storId, expectedStorage)
       _ <- permissionDsl.addPermissions(Permission(storName, "read"), Permission(storName, "write"))
+      _ <- log("Creating first storage")
       _ <- storagesDsl.createStorage(payload2, projectRef)
+      _ <- log("Created second storage")
       _ <- storagesDsl.checkStorageMetadata(projectRef, storageId2, expectedStorageWithPerms)
     } yield succeed
   }
 
+  def log(msg: String): IO[Unit] = IO.println(msg)
```

Comment:
To remove ?

Reply:
Can do, I find both the tests and production code could do with more logs 😅